doc/api.tex

```diff
@@ -16,8 +16,11 @@
         feed them with \texttt{FIELD} values from \texttt{TABLE} rows or from \texttt{PARAM}.
  \end{enumerate}
 
-These different levels should be hidden to the end user. Using MIVOT annotations makes sense only if the client code provides 
+Level 1 and 2 may only be useful for client developpers, they should be hidden to the end user. 
+Using MIVOT annotations makes sense only if the client code provides 
 the users with a direct access to both modeled quantities and links that connect them. 
+This can be achieved by providing level 3 and 4 APIs to end users.
```

Comment:
Yes, this looks better. I would say that MIVOT is a machine level system that relies primarily on higher level APIs to provide representations of the models (level 3 and 4) to end users

Reply:
I've changed the text with your wording